MoltenVK/MoltenVK/Utility/MVKEnvironment.h

```diff
@@ -166,6 +166,17 @@
 #   define MVK_CONFIG_AUTO_GPU_CAPTURE_SCOPE    	MVK_CONFIG_AUTO_GPU_CAPTURE_SCOPE_NONE
 #endif
 
+/**
+ * The file to capture automatic GPU traces to, instead of capturing to Xcode. This is
+ * useful when trying to capture a one-shot trace, but the program cannot be run under
+ * Xcode's control. Tilde paths may be used to place the trace document in a user's home
+ * directory. This functionality requires macOS 10.15 or iOS 13. If left blank, automatic
+ * traces will be captured to Xcode.
+ */
```

Comment:
```suggestion
/**
  * The file to capture automatic GPU traces to, instead of capturing to Xcode. This is
  * useful when trying to capture a one-shot trace, but the program cannot be run under
  * Xcode's control. Tilde paths may be used to place the trace document in a user's home
  * directory. This functionality requires macOS 10.15 or iOS 13. If left blank, automatic
  * traces will be captured to Xcode.
  */
```
Can you add a suitable description in `vk_mvk_moltenvk.h` as well, following the description patterns there, please? Perhaps right after the description of `MVK_CONFIG_AUTO_GPU_CAPTURE_SCOPE`?